server/liveblog/tests/environment.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from apps.io.tests import setup_providers
+from superdesk import tests
+from superdesk.tests import setup_auth_user
+from superdesk.tests.environment import setup_search_provider
+from superdesk.vocabularies.command import VocabulariesPopulateCommand
+from liveblog.tests import setup_auth_consumer, test_consumer
+
+
+def setup_before_scenario(context, scenario, config, app_factory):
```

Comment:
Could you use `setup_before_scenario` from `superdesk` and add your modifications without duplicate all other general parts